test/framework/src/main/java/org/elasticsearch/test/junit/listeners/ReproduceInfoPrinter.java

```diff
@@ -82,7 +82,7 @@ public void testFailure(Failure failure) throws Exception {
         gradleMessageBuilder.appendAllOpts(failure.getDescription());
 
         //Rest tests are a special case as they allow for additional parameters
-        if (ESRestTestCase.class.isAssignableFrom(failure.getDescription().getTestClass())) {
+        if (ESClientYamlSuiteTestCase.class.isAssignableFrom(failure.getDescription().getTestClass())) {
```

Comment:
I think we have to make sure that -Dtests.rest.cluster gets printed out also when ESRestTestCase fails?